bee-message/tests/address.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message::prelude::*;
+use bee_packable::Packable;
+
+use core::convert::TryInto;
+
+const ED25519_ADDRESS: &str = "52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c649";
+
+// The kind of an `Address` is the kind of the underlying address.
+#[test]
+fn kind() {
+    let bytes: [u8; 32] = hex::decode(ED25519_ADDRESS).unwrap().try_into().unwrap();
```

Comment:
maybe we could have a helper function that decodes arrays into arrays

Reply:
I'll put one in `util` 🙂

Reply:
Ignore me 🤦